core/main.go

```diff
@@ -76,15 +76,31 @@ func stopRunningServices(sdk *enginesdk.SDK) error {
 	return errs.ErrorOrNil()
 }
 
+func loadOrGenConfigAccount(kb *cosmos.Keybase, cfg *config.Config) (keys.Info, error) {
```

Comment:
this could be in engine keybase package to extend cosmos keybase.

Reply:
All the function you mean?
I already put the `Exist` function in `keybase`.
I don't think we will have this logic of `loadOrGen` in any other place.

Reply:
> All the function you mean?

Yes

>  I don't think we will have this logic of loadOrGen in any other place.

And that's fine. The Keybase wrapper was created to handle such cases. so you don't put them in other places (like main.go). 

```go
func (kb *Keybase) LoadOrGenAccount(name, password string) (keys.Info, error) { 
}
```

Reply:
This is not compatible with Anthony's suggestion https://github.com/mesg-foundation/engine/pull/1459/files#r342436168